great_expectations/checkpoint/checkpoint.py

```diff
@@ -99,6 +106,85 @@ def get_substituted_config(
                     self._substituted_config = substituted_config
                 return substituted_config
 
+    def get_runtime_batch_request(
+        self,
+        substituted_runtime_config: CheckpointConfig,
+        validation_batch_request: Union[dict, BatchRequest, None],
```

Comment:
Perhaps use Optional[dict, BatchRequest] -- more terse by not having "None" in there...

Reply:
Yep - changing.